tests/e2e/scripts/init.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Script to install pre-requisites on vsphere node to run e2e test
+
+PATH=$PATH:/usr/local/go/bin/:/usr/local/bin
+echo "Installing Kubectl"
```

Comment:
```suggestion
echo 'Installing Kubectl'
```

Use double quotes only when something is to be expanded, not for static strings.